src/vmm/src/rpc_interface.rs

```diff
@@ -325,6 +319,26 @@ impl<'a> PrebootApiController<'a> {
             CreateSnapshot(_) | SendCtrlAltDel => Err(VmmActionError::OperationNotSupportedPreBoot),
         }
     }
+
+    #[cfg(target_arch = "x86_64")]
+    fn load_snapshot(&mut self, load_params: LoadSnapshotParams) -> ActionResult {
+        let load_start_us = utils::time::get_time_us(utils::time::ClockType::Monotonic);
+
+        let loaded_vmm = persist::load_snapshot(
+            &mut self.event_manager,
+            &self.seccomp_filter,
+            load_params,
+            VERSION_MAP.clone(),
+        );
+
+        let delta_us = utils::time::get_time_us(utils::time::ClockType::Monotonic) - load_start_us;
+        METRICS.snapshot.load_us.store(delta_us as usize);
+        info!("Loading the snapshot took {} us.", delta_us);
```

Comment:
`us` is a widely accepted simplification for `μs`. `microseconds` is too verbose in this message IMO and I wouldn't change it with `μs` since this may involve Unicode issues for some editors I guess :-?.